src/Controller/MailslurpController.php

```diff
@@ -0,0 +1,42 @@
+<?php
+namespace App\Controller;
+
+use MailSlurp\Apis\InboxControllerApi;
+use MailSlurp\Apis\WaitForControllerApi;
+use MailSlurp\Configuration;
+use MailSlurp\Models\SendEmailOptions;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\HttpFoundation\Request;
+
+class MailslurpController
+{
+    public $config;
+
+    public function setUp()
+    {
+        if( $this->config == null){
+            $this->config = Configuration::getDefaultConfiguration()->setApiKey('x-api-key','d7eec2cf7d3241701019794d2666599601602b4f590e79358714e6038fb22707');
+        }
+        return $this->config;
+    }
+
+    public function newEmail(Request $request):Response
+    {
+        // create inbox and waitFor controllers
+        $inbox_controller = new InboxControllerApi(null, $this->setUp());
```

Comment:
hay un "hack" que se usa en PHP para saber qué se está pasando a las funciones, por ejemplo en lugar de:

`InboxControllerApi(null, $this->setUp())`

se puede utilizar:

`InboxControllerApi($param1 = null, $config = $this->setUp())`

para que sepamos qué es cada parámetro, donde $param1 es el nombre del parámetro que espera la función